app/views/security_group/edit.html.haml

```diff
@@ -1,4 +1,10 @@
-%form#form_div{:name => "angularForm", 'ng-controller' => "securityGroupFormController"}
+%form#form_div{:name           => "angularForm",
+   "ng-controller" => "securityGroupFormController as vm",
+   "miq-form"      => true,
+   "model"         => "vm.securityGroupModel",
+   "model-copy"    => "vm.modelCopy",
```

Comment:
Can you add one more entry here?
```ruby
"form-changed" => true,
```

`form-changed` is a new directive that does dirty-checking on the entire form.
Pretty soon, we would be deprecating the `checkchange` directive that is applied on every form element.
So when you use `form-changed`, be sure to remove all instances of `checkchange`